intervention_compassion/mappings/intervention_mapping.py

```diff
@@ -0,0 +1,147 @@
+            ('field_office_id.name', 'compassion.field.office'),
+        "GlobalPartner_Name": None,
+        "CenterBasedMinimumAge": None,
+        "HomeBasedMaximumAge": None,
+        "HomeBasedMinimumAge": None,
+        "ActualCostPerBeneficiaryUSD": None,
+        "ActualLocalContribution": 'local_contribution',
+        "EstimatedCostPerBeneficiaryUSD": None,
+        "EstimatedLocalContribution": None,
+        "EstimatedLocalContributionUSD": 'estimated_local_contribution',
+        "TotalActualCosts": None,
+        "TotalActualCostsUSD": 'total_cost',
+        "TotalEstimatedCost": 'estimated_costs',
+        "TotalEstimatedCostsUSD": None,
+        # generic
+        "InterventionCategory_Name":
+            ('category_id.name', 'compassion.intervention.category'),
+        # generic
+        "InterventionSubCategory_Name":
+            ('subcategory_id.name', 'compassion.intervention.subcategory'),
+        "ActualDurationInMonthsNbr": None,
+        "ActualEndDate": 'end_date',
+        # "AdditionalFundsCommitted": False,  # ???
+        "AdditionalMarketingInformation":
+            'additional_marketing_information',
+        "AllocatedSurvivalSlots": None,
+        "ApprovedDate": None,
+        "ApprovedSLACostUSD": 'approved_sla_costs',
+        # "AssociatedInactiveICPCount": 0,  # ???
+        "BackgroundInformation": 'background_information',
+        "CancellationReason": 'cancel_reason',
+        "CognitiveVocationalActivities": None,
+        "ContractorInformation": None,
+        "CurrentPlannedEndDate": 'planned_end_date',
+        "Description": 'description',
+        "DesiredCommuniyBenefits": None,
+        "EstimatedImpactedBeneficiaryQuantity":
+            'estimated_impacted_beneficiaries',
+        "ExpectedDurationMonths": 'start_date',
+        # "FieldOfficeApprover": "ciITC1test5 ciITC1test5",  # ???
+        "FirstTimeMothersAverageAge": None,
+        "FiscalYear": None,
+        "FocusAreaNumber": None,
+        "FundCode": None,
+        "FundingGlobalPartners": 'funding_global_partner_ids',
+        "FundingStatus": 'funding_status',  # was "Fully Held", not known
+        "FundingStatusCommittedDate": None,
+        # "GlobalPartnerRequestedDeliverables": 'deliverable_ids',  # ???
+        "GlobalPartnerSLA": None,
+        "HouseholdChildrenAverageQuantity": None,
+        "ICP": 'icp_id',  # generic, not sure
+        "Intervention_ID": 'intervention_id',
+        "ImpactedBeneficiaryQuantity": 'impacted_beneficiaries',
+        "ImplementationRisks": 'implementation_risks',
+        "InitialPlannedEndDate": 'initial_planned_end_date',
+        "InitiationType": None,
+        # "IsApproved": False,  # ???
+        "IsFieldOfficePriority": 'is_fo_priority',
+        # "IsICPSuspendedTerminated": False,  # ???
+        # "IsIncludedInInterventionStrategy": False,  # ???
+        "LeadershipApproval": None,
+        "LinkToFieldOfficeStrategy": None,
+        "LocalContributionUSD": None,
```

Comment:
ok, so I will set "ActualLocalContribution" to None, (see line 38)
Otherwise 'local_contribution' would be mapped to both "ActualLocalContribution" and "LocalContributionUSD"